pkg/sql/storageparam/tablestorageparam/table_storage_param.go

```diff
@@ -162,24 +155,6 @@ var tableParams = map[string]tableParam{
 			return nil
 		},
 	},
-	`ttl_automatic_column`: {
```

Comment:
you'll need to keep this for backwards compat. send a notice when this is set that this is now a no-op.

Reply:
Added back with a notice.